e3db/config.py

```diff
@@ -163,6 +173,8 @@ def write(self, profile=''):
                 'client_email': str(self.client_email),
                 'public_key': str(self.public_key),
                 'private_key': str(self.private_key),
+                'public_signing_key': str(self.public_signing_key),
+                'private_signing_key': str(self.private_signing_key),
```

Comment:
Readme should be updated showing instantiating a client with signing keys

Reply:
https://github.com/tozny/e3db-python#loading-configuration-and-creating-a-client

Reply:
I also updated backup and example sections to include signing keys. Assuming they are necessary for backups?

Reply:
Perfect!